### PR TITLE
[Vue] Fix: this.userManagementService is not a function

### LINKED ISSUE #17565: Vue client: userManagementService is not provided
##### **Overview of the issue**

For Vue client, userManagementService is not provided for admin/user-management.

##### **Motivation for or Use Case**

Runtime errors occur in admin/user-management menu.

##### **Reproduce the error**

Create a new app as follow:
```
? Which *type* of application would you like to create? Monolithic application (recommended for simple projects)
? What is the base name of your application? usermanagement
? Do you want to make it reactive with Spring WebFlux? No
? What is your default Java package name? com.mycompany.myapp
? Which *type* of authentication would you like to use? JWT authentication (stateless, with a token)
? Which *type* of database would you like to use? SQL (H2, PostgreSQL, MySQL, MariaDB, Oracle, MSSQL)
? Which *production* database would you like to use? PostgreSQL
? Which *development* database would you like to use? H2 with disk-based persistence
? Which cache do you want to use? (Spring cache abstraction) Ehcache (local cache, for a single node)
? Do you want to use Hibernate 2nd level cache? Yes
? Would you like to use Maven or Gradle for building the backend? Maven
? Do you want to use the JHipster Registry to configure, monitor and scale your application? No
? Which other technologies would you like to use? 
? Which *Framework* would you like to use for the client? Vue
? Do you want to generate the admin UI? Yes
? Would you like to use a Bootswatch theme (https://bootswatch.com/)? Default JHipster
? Would you like to enable internationalization support? Yes
? Please choose the native language of the application English
? Please choose additional languages to install 
? Besides JUnit and Jest, which testing frameworks would you like to use? 
? Would you like to install other generators from the JHipster Marketplace? No
```

##### **Suggest a Fix**

New UserService added to entities.
But admin/user-management still use userManagementService.

##### **JHipster Version(s)**

7.5.0

##### **JHipster configuration**

.yo-rc.json
```
{
  "generator-jhipster": {
    "applicationType": "monolith",
    "baseName": "usermanagement",
    "jhipsterVersion": "7.5.0",
    "skipUserManagement": false,
    "skipCheckLengthOfIdentifier": false,
    "skipFakeData": false,
    "jhiPrefix": "jhi",
    "entitySuffix": "",
    "dtoSuffix": "DTO",
    "testFrameworks": [],
    "blueprints": [],
    "otherModules": [],
    "pages": [],
    "creationTimestamp": 1642147543820,
    "serviceDiscoveryType": "no",
    "reactive": false,
    "authenticationType": "jwt",
    "packageName": "com.mycompany.myapp",
    "serverPort": "8080",
    "cacheProvider": "ehcache",
    "enableHibernateCache": true,
    "databaseType": "sql",
    "devDatabaseType": "h2Disk",
    "prodDatabaseType": "postgresql",
    "buildTool": "maven",
    "serverSideOptions": [],
    "websocket": false,
    "searchEngine": false,
    "messageBroker": false,
    "enableSwaggerCodegen": false,
    "clientFramework": "vue",
    "withAdminUi": true,
    "clientTheme": "none",
    "enableTranslation": true,
    "nativeLanguage": "en",
    "jwtSecretKey": "NTQzN2UzNWE4NTEzMjhjODRmMTU4NDU3Yjg3NWMwNjI4YjE3MzY3MThlMWM1YWU3Yzk0ZTUwNDQyMTNjNzczZWI2YzdiNGY5MDVjNzM1YThjZjVkODk4YjA5MTY2MTExNDFiMDFhMWFlN2Q5NDNjMzNjZmU2NDJjMDhiODc1MTE=",
    "devServerPort": 9060,
    "clientPackageManager": "npm",
    "clientThemeVariant": "",
    "languages": ["en"],
    "enableGradleEnterprise": false
  }
}
```

- [x] Checking this box is mandatory (this is just to show you read everything)

### PR DESCRIPTION
**Pull request by qmonmert**
Fix #17565

---

Please make sure the below checklist is followed for Pull Requests.

- [x] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [x] Tests are added where necessary
- [x] The JDL part is updated if necessary
- [x] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [x] Documentation is added/updated where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (below reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

**Comment by mshima** [comment]
Is there a problem with cypress tests or with samples?
It was supposed to test this use case.

**Comment by qmonmert** [comment]
@mshima bug with simple app https://github.com/jhipster/generator-jhipster/issues/17565

File: generators/client/templates/vue/src/main/webapp/app/main.ts.ejs [hunk 1]
```diff
@@ -99,9 +100,10 @@ new Vue({
   router,
   provide: {
     loginService: () => loginService,
-<%_ if (!skipUserManagement) { _%>
+    <%_ if (!skipUserManagement) { _%>
```
Comment (mshima) [inline generators/client/templates/vue/src/main/webapp/app/main.ts.ejs:103]:
```suggestion
<%_ if (!skipUserManagement) { _%>
```

**Review by mshima — COMMENTED** [review]
Only page visit is implemented at cypress tests.
The PR should fix the error, but I think a basic regression test would be nice.

**Comment by qmonmert** [comment]
@mshima are you ok with that?

**Pull request merged**